Modelica/Fluid/Examples/AST_BatchPlant.mo

```diff
@@ -1940,7 +1940,7 @@ Implemented trace substances and missing equation for outflow of multi substance
               {-20,-30},{-20,-21},{-22,-21}}, color={0,127,255}));
       connect(pipe.port_a, tank1.ports[2]) annotation (Line(points={{40,0},{40,-28},
               {-18,-28},{-18,-20},{-18,-21}}, color={0,127,255}));
-      annotation (experiment(StopTime=35));
+      annotation (experiment(StopTime=35, Tolerance=1e-10));
```

Comment:
Is it really sane that a modell needs this sort of tolerance?

Reply:
Would it make more sense to just apply this when generating the reference?

Reply:
We could use 1e-7 instead if that's more acceptable.
The relative error in #3782 is about 1e-6.

Reply:
That sounds like a more reasonable number to me.

The relative error can be measured in different ways, and I'm sure one can write essays on how to compare temperatures in various domains.  The way we measure this error at Wolfram corresponds to what one can see in the figure (not using a dynamic range that goes all the way down to zero), and we currently use a relative tolerance of 8e-2 to let the test through.

It would be interesting to see what the corresponding number would be at 1e-7 solver tolerance.  As always, it would have been so much better if the model change was accompanied by an updated reference result in the same PR.  Then I would have been able to accept the PR instead of just leaving comments and hoping for the best once the updated reference arrives.

Reply:
This has now been addressed.  Resolving.